core/domain/stats_jobs_one_off.py

```diff
@@ -303,6 +323,23 @@ def reduce(exp_id, values):
 
     @classmethod
     def prepare_reduce(cls, exp_id, values):
+        """Runs the reduce step to extract aggregate exploration statistics from
+        the dicts passed in by the mapping function.
+
+        Args:
+            exp_id: str. The exploration id.
+            values: list(dict(str, str)). List of dicts of events for preparing
+                the reduce.
+
+        Returns:
+            tuple(list(dict), list(ExplorationStats), list(str)). 3-tuple where:
+            - The first element is the list of dicts of different version-wise
```

Comment:
> For description about the dict: The keys are dependent on version numbers like: num_starts_v2, num_completions_v2, ... etc. Also, there are around 10 keys like this and it doesn't make much sense to me. Shall I write them here and how to handle versions in keys?

Are the keys a fixed set? I think so, in which case we should document them the usual way. I think the dict will always include v1 and v2 keys (but please check).

For each key, specify the type of its corresponding value, and a description of that value. You should be able to get some context by looking at how the result of this job is used. This class might also be helpful: https://github.com/oppia/oppia/blob/develop/core/domain/stats_domain.py#L53

Reply:
Yes, the keys are fixed in this case and are separate for v1 and v2. I've added them. Thanks!